package-scripts/datadog-agent/preinst

```diff
@@ -34,10 +34,15 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
 
     # Prepare the GPG keys rotation:
     # Add the new one to the list of trusted APT keys.
+    # Some servers are on severely resticted networks.  Check for proxy,
+    # and check if key already exists.
```

Comment:
One last thing:
- there's a typo `resticted` -> `restricted`
- could you update the comment? (i.e. remove or clarify `Check for proxy` since the script itself doesn't check for proxy anymore :) )